src/css/BaseStyle.js

```diff
@@ -66,12 +69,27 @@ export type BaseStyle = {|
   backgroundPosition?: props.backgroundPosition,
   'background-position'?: props.backgroundPosition,
 
+  backgroundPositionX?: props.backgroundPositionX,
+  'background-positionX'?: props.backgroundPositionX,
+
+  backgroundPositionY?: props.backgroundPositionY,
+  'background-positionY'?: props.backgroundPositionY,
+
   backgroundRepeat?: props.backgroundRepeat,
   'background-repeat'?: props.backgroundRepeat,
 
+  backgroundRepeatX?: props.backgroundRepeatX,
+  'background-repeat-x'?: props.backgroundRepeatX,
+
+  backgroundRepeatY?: props.backgroundRepeatY,
+  'background-repeatY'?: props.backgroundRepeatY,
```

Comment:
`background-repeatX` and `background-positionX` should really be `background-repeat-x`, same with y